### ISSUE TITLE
Understanding document 1.4.10: examples instead of benefits

### ISSUE THREAD

**Issue by gjccopinga** [w3c/wcag#3428]
Hi,

If I look at the understanding document on SC 1.4.10 (https://www.w3.org/WAI/WCAG21/Understanding/reflow) and go to the "Benefits" section I do not see benefits (for people with disabilities) but I see some examples. Please add the benefits here and move the examples to the "Examples" section.

Gerard

**Comment by detlevhfischer** [comment]
@gjccopinga There is a Benefits section https://www.w3.org/WAI/WCAG21/Understanding/reflow#benefits mentioning the main benefit, being able to read content with text enlarged without having to scroll sideways. Do you think more benefts should be added?

**Comment by gjccopinga** [comment]
I saw the benefits section. But I did not recognize the two 'benefits' mentioned there as benefits. they 'looked' like examples. Probably because of the presentation with the bold text in the first sentence. That (bold) presentation is not being used in the previous succescriteria (1.4.x). And the wording is kind of different than usual (see also previous succescriteria 1.4.x). If I look closer I see that people wit disabilities are being mentioned at the end of the second 'benefit' and users in general are being mentioned in the first 'benefit'.
So the problem for me is probably more that the presentation is not consistent with the other succescriteria and the wording is also not very consistent (in comparison with the benefits sections of other succescriteria).

You can close the issue. But maybe it is a good idea to change the presentation and the wording. To make it more consistent.

**Comment by alastc** [comment]
It is a little inconsistent with other understanding docs, so we can leave open. Not high priority though, the core information is there.
